### ISSUE TITLE
Import/export pages/templates to implement Dataflow from ERP

### ISSUE THREAD

**Issue by Stefan-Lammers** [elementor/elementor#10360]
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
The task at hand is: we will have maybe a thousand Pages with Texts and pictures. These my change at any time, according to changes in the ERP System of the client. These changes have to be promoted to the wordpress-pages - without manual work. We plan to use WP ALL IMPORT/EXPORT, but are open for any other implementation.

I can see two solutions for problem:

I could export all templates via "my templates", change the content (via php script) and reimport the templates. But: newly imported templates have different (new) IDs, so, they are not connected to the pages anymore. I WOULD WELCOME A WAY TO OVERWRITE EXISTING TEMPLATES ON IMPORT.

Second solution: I could export all templates/pages via WP ALL EXPORT into CSV, change the content, delete all templates (to have changed templates/pages with identical IDs) and reimport via WP ALL IMPORT. But: I have no idea wich datafield contains the JSON String. I NEED a WAY OR DOCUMENTATION TO CHANGE THE JSON DATA?

Alternative
If the data is stored someplace else in the database i would greatly appreciate info on that - reading data out of databases and manipulating it is my daily bussiness, so that would be (hopefully) the most easy way? THEN I WOULD NEED DOCUMENTATION ON THE DATABASE

I presented these Problems/Ideas to the support, but they regard this issue as being (more of) a feature Request, so here it ist, and I hope hear from a develope soon (and of course we would be willing to put in some work).

**Comment by Stefan-Lammers** [comment]
Ok, guys, nevermind - i found an easy way to do it the other way round (not poking data into elementor but asking for data from elementor). Would like to above mentioned functions anyway, but have no direct need right now :)

**Comment by Soetens** [comment]
How did you solve this? I'm running into the same problem.

**Comment by Stefan-Lammers** [comment]
in short:

I programmed a php-script to break down the ERP Data into managable bits. This script is triggerd by ERP whenever there are new data.

on the other hand, i installed a plugin for PHP Snippets - those snippets reads the manable bits, put them together as defined for the snippets purpose and put them into a html-template (I created a dozend or so different templates as needed by the page).

then i made the pages with elementor as normal (with wide areas only populated by shortcodes to those snippets. At some places that made a bit of „magic“ necessary, like having a arccordeon with one element, that element is made invisible by a javascript in above mentioned html template and then (the accordeon) is populated by that snippet with new elements, like needed from analysis of the „managable bits“ from the ERP data.

so, some steps needed, no tool to make this easy, but it works and it works just fine and reasonable fast.

Liebe Grüße
Stefan
